package.json

```diff
@@ -87,7 +87,7 @@
     "storybook-addon-jsx": "^5.3.0"
   },
   "dependencies": {
-    "danger": "^7.0.14",
+    "danger": "^6.1.13",
```

Comment:
Do we want to do this or change the peerDependencies on duti to `danger@^7.0.0`?

Reply:
I'd be wary of doing anything on duti until Sam gets back